docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,83 @@
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+In this tutorial, you will analyze activity on a popular news aggregation website. Similar to your data pipelines, you will fetch data from the website, clean it up, and build a report that summarizes some findings. You'll then tell Dagster to occassionally update the data and the report, which Dagster calls _assets_.
+
+---
+
+## Introduction to Software-defined assets
+
+Before building a Dagster project, you'll first learn about Dagster's core concept: the Software-defined asset (SDA).
+
+An asset is an object in persistent storage that captures some understanding of the world. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like Amazon S3
+- A machine learning model
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you will:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Start to think about what parts of your current pipeline are assets
+
+### Parts of a Software-defined asset
+
+Software-defined assets wrap objects to enable data engineers to do more with them.
+
+As a Dagster user, you're able to build assets in code. In the sample asset below, the data frame returned is an asset and Dagster recognizes that the function is an SDA 
+
+```
+@asset
+def hackernews_topstories(hackernews_topstory_ids):
+    results = []
+    for item_id in hackernews_topstory_ids:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    return pd.DataFrame(results)
+```
+
+### Building a DAG of assets
+
+Defining multiple assets creates a DAG (directed acyclic graph) of what assets depend on other assets. This DAG, viewable in Dagster's web-based UI, helps you to:
+
+- Understand how your assets relate to each other
+- Empower you and your teammates to act, learn, and debug your pipelines
+
+### Extending with ops and jobs
+
+Asset-based orchestration may seem like a new way to approach data orchestration, but it is similar to traditional task-based orchestration. Under the hood, assets are a wrapper around Dagster's task-based framework. Therefore, all work done with SDAs can also done with tasks. You can even use multiple Ops to create an asset.
+
+Dagster's ideology believes that data engineers should use assets. However, users can use Dagster's Ops and Jobs' task-based framework when assets are not the right fit. Some common situations to use the task-based framework are when you are:
```

Comment:
this phrase is slightly funny.  "Dagster's ideology believes that data engineers should use assets."

would suggest

"We think that SDAs enable the best long-term approach to building and maintaining pipelines. Add some commentary about how assets save you time because of X and Y"

Reply:
+1 ^^

Reply:
god bless, thank you. I think I had 3 different iterations on this sentence but each of them felt tense, terse, or straight up draconian.